info.toml

```diff
@@ -624,6 +624,38 @@ Try mutating the incoming string vector.
 Vectors provide suitable methods for adding an element at the end. See 
 the documentation at: https://doc.rust-lang.org/std/vec/struct.Vec.html"""
 
+# Generics
+
+[[exercises]]
+name = "generics1"
+path = "exercises/generics/generics1.rs"
+mode = "compile"
+hint = """
+Vectors in rust make use of generics to create dynamically sized arrays of any type.
+You need to tell the compiler what type we are pushing onto this vector."""
+
+[[exercises]]
+name = "generics2"
+path = "exercises/generics/generics2.rs"
+mode = "test"
+hint = """
+Think carefully about what we need to do here. Currently we are wrapping only values of
+type 'u32'. Maybe we need to update the explicit references to this data type somehow?
+"""
+
+[[exercises]]
+name = "generics3"
+path = "exercises/generics/generics3.rs"
+mode = "test"
+hint = """
+To find the best solution to this challenge you're going to need to think back to your 
```

Comment:
I'm not 100% sure about this exercise. It does require knowledge of traits (which is the section before this one), it seems kind of logical to me as the areas are quite related, but I don't know how insulated you like the exercises to be. Maybe it could be a candidate to be extended slightly and made into a new `test`?

Reply:
I'd say use it for now and readjust it if it turns out to not work that well!